examples/cookies.rs

```diff
@@ -0,0 +1,28 @@
+#![feature(async_await, futures_api)]
+
+use cookie::Cookie;
+use tide::{cookies::CookiesExt, middleware::CookiesMiddleware, Context};
+
+/// Tide will use the the `Cookies`'s `Extract` implementation to build this parameter.
```

Comment:
Now that there is no `Extract` trait, shouldn't this change? Something in the lines of:

```
/// Tide will use the `CookiesExt` extension trait to extract the cookie from the request.
```